.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+**Pull Request Title**
+Please provide an appropriate title for this pull request.
+
+**Related Issue(s):**
+Add `closes #issue_number` if your pull request is closing an issue. Write `N/A` if it is not closing any issue. But in that case, we recommend you to create an issue first, as it helps us to keep track of the codebase.
+
+**Describe your change**:
+
+* [ ] Add an algorithm?
+* [ ] Fix a bug or typo in an existing algorithm?
+* [ ] Documentation change?
+
+**Checklist**:
+
+* [ ] I have read and followed the [contributing guidelines](/CONTRIBUTING.md).
```

Comment:
Looks good, but we need a `CONTRIBUTING.md` file beforehand. Can you please create an issue and work on that before this PR?